src/applications/mhv/secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -608,77 +598,51 @@ const ComposeForm = props => {
+                className="
                   vads-u-border-top--1px
                   vads-u-padding-top--3
                   vads-u-margin-top--3
                   vads-u-margin-bottom--neg2"
-                >
-                  <BlockedTriageGroupAlert
-                    blockedTriageGroupList={blockedTriageGroupList}
-                    alertStyle={BlockedTriageAlertStyles.ALERT}
-                    parentComponent={ParentComponent.COMPOSE_FORM}
-                  />
-                </div>
-              ))}
+              >
+                <BlockedTriageGroupAlert
+                  blockedTriageGroupList={blockedTriageGroupList}
+                  alertStyle={BlockedTriageAlertStyles.ALERT}
+                  parentComponent={ParentComponent.COMPOSE_FORM}
+                />
+              </div>
+            )}
 
-          {mhvSecureMessagingBlockedTriageGroup1p0
-            ? recipientsList &&
-              (!noAssociations &&
-                !allTriageGroupsBlocked && (
-                  <>
-                    <VaSelect
-                      enable-analytics
-                      id="recipient-dropdown"
-                      label="To"
-                      name="to"
-                      value={selectedRecipient}
-                      onVaSelect={recipientHandler}
-                      class="composeSelect"
-                      data-testid="compose-recipient-select"
-                      error={recipientError}
-                      data-dd-privacy="mask"
-                      data-dd-action-name="Compose Recipient Dropdown List"
-                    >
-                      {sortRecipients(recipientsList)?.map(item => (
-                        <option key={item.id} value={item.id}>
-                          {item.name}
-                        </option>
-                      ))}
-                    </VaSelect>
-                  </>
-                ))
-            : recipientsList && (
-                <>
-                  <VaSelect
-                    enable-analytics
-                    id="recipient-dropdown"
-                    label="To"
-                    name="to"
-                    value={selectedRecipient}
-                    onVaSelect={recipientHandler}
-                    class="composeSelect"
-                    data-testid="compose-recipient-select"
-                    error={recipientError}
-                    data-dd-privacy="mask"
-                    data-dd-action-name="Compose Recipient Dropdown List"
-                  >
-                    {sortRecipients(recipientsList)?.map(item => (
-                      <option key={item.id} value={item.id}>
-                        {item.name}
-                      </option>
-                    ))}
-                  </VaSelect>
-                </>
-              )}
+          {recipientsList &&
+            !noAssociations &&
+            !allTriageGroupsBlocked && (
+              <>
+                <VaSelect
```

Comment:
Can this be updated to v3 by adding the `uswds` prop?

Reply:
@micahchiang is it alright if we pull this work in a separate effort? We are hoping that all `uswds` transition changes are tested properly in its own scope

Reply:
You can, but you should be aware that V3 components are becoming opt out only soon, and it'll be your team's responsibility to ensure your application continues to function as normal. [More information in slack](https://dsva.slack.com/archives/C03R5SBELQM/p1707929381167999)